modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/page/change/DDMFormPageChangeTracker.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.form.page.change;
+
+/**
+ * @author Bruno Oliveira
+ * @author Carolina Barbosa
+ */
+public interface DDMFormPageChangeTracker {
+
+	public DDMFormPageChange getDDMFormPageChangeById(String ddmFormInstanceId);
```

Comment:
@brunofariasdeo, please rename the method to `getDDMFormPageChangeByDDMFormInstanceId` to keep consistency with the parameter renaming.